src/Hosting/Hosting/src/Internal/HostingLoggerExtensions.cs

```diff
@@ -11,28 +11,23 @@
 
 namespace Microsoft.AspNetCore.Hosting
 {
-    internal static class HostingLoggerExtensions
+    internal static partial class HostingLoggerExtensions
```

Comment:
Do we want to standardize on nested log classes while we're at it? See RequestDelegateFactory.Log

Reply:
Yes

Reply:
The application error / startup error appear in 3 different places. I'm leaving those as-is because it's repetitive and there's actually useful code that is executed prior to the logging (the exception gets unwrapped). Apart from that individual log messages were moved to nested classes.